runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -2731,7 +2731,7 @@ void set_up_for_polling(void)
     uint32_t cq_mode = GNI_CQ_NOBLOCK;
 
     polling_task_blocking_cq = chpl_env_rt_get_bool("COMM_UGNI_BLOCKING_CQ",
-                                                    false);
+                                                    polling_task_blocking_cq);
```

Comment:
I would actually prefer `true` here and no initializer on the variable decl.  The latter is unneeded unless there's a path through execution that doesn't pass through this `getenv` call, and the former would make the default easier for a reader to see.  Otherwise you have to go backwards 1400 lines to find it.

Reply:
Yeah, good call. I was originally thinking -- "Oh, it's nice to see the default at the declaration site so you don't have to go through the code to find it", but in hindsight I think you're right.